tests/format_tests.py

```diff
@@ -0,0 +1,36 @@
+import unittest
+from pycpfcnpj import cpf, cnpj, mask
+
+
+class FORMATTests(unittest.TestCase):
```

Comment:
Use PEP8 naming format. FormatTests, for instance. However, it's not a clear naming. What do you think about MaskingTests?